moderatorguide.md

```diff
@@ -3,62 +3,62 @@
+* monitor participant behavior in the chat for breaches of the code of conduct
+* give a closing remarks
 
-Below we provide more detailed information on certain aspects of the moderator role. 
+Below we provide more detailed information on certain aspects of the moderator role. Note that the moderator role requires full concentration, lot of focus and continuous monitoring of multiple channels. In that sense, moderating an online event is significantly more demanding than the role of a chair in an offline conference.
 
 ## How to
 
 ### Opening and closing remarks
 
 At the beginning of the seminar: 
 
-* highlight the speaker’s screen-share,
-* introduce the speaker (name, current institution, something about the talk or their current work),
-* unmute the speaker and invite them to begin their seminar.
+* ensure that everyone is muted and cannot unmute themselves by clicking "mute all participants" and unchecking "participants may unmute themselves" checkbox
+* announce that the talk is recorded and start the recording
+* deliver the opening remarks, introduce VSF and the speaker
+* turn on spotlight of the speaker’s webcam stream
+* unmute the speaker and invite them to begin their seminar
 
 At the end of the seminar: 
 
 * thank the speaker 
 * initiate the post-talk question/answer period
-* stop highlighting speaker's screen-share.
+* stop highlighting speaker's screen-share
 
-NB: a virtual conference cannot have applause after the seminar ends! However, Zoom does have a clapping hands image that audience members may put up next to their name in the participants list. 
+NB: a virtual conference cannot have applause after the seminar ends! However, you may invite the audience to use for example the Zoom "applaud" emote.
 
 
 ### Question guidelines
 
-Unless otherwise agreed by the organizers, the following guidelines apply:
-
-* Short talks: after the talk ends.
-* Long talks: questions may be asked at any time.
+Our suggested guidelines for when questions are allowed:
+* short talks: after the talk ends
+* long talks: questions may be asked at any time
 
 Mediating questions:
 
-* Monitor the participants list to observe attendees signaling for questions (“Raise hand option”).
-* Let speaker and audience know that there is a question, and signal the asker by stating their name and unmuting their microphone.
-* Leave the attendee unmuted while interacting with the speaker.
-* Mute the attendee when finished, and “lower their hand”.
-* In the event of a prolonged discussion, you may get the talk back on track by suggesting that the conversation continue separately after the seminar.
+* monitor the participants list to observe attendees signaling for questions (“Raise hand” Zoom emote)
+* let speaker and audience know that there is a question, and signal the asker by stating their name and unmuting their microphone
+* leave the attendee unmuted while interacting with the speaker
+* mute the attendee when finished, and “lower their hand”
```

Comment:
```suggestion
* leave the asker unmuted while interacting with the speaker
* mute the asker when finished and “lower their hand”
```
Should keep the name of "asker" or "attendee" consistent on these bullet points. I like "asker" but I think either is ok.